src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,204 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+        function commonParams = getCommonAFEParams( )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...   % default is 0
+                'pp_intTimeSecRMS', 4, ...      % default is 500E-3
+                'pp_bBinauralRMS', true, ...    % default is true
+                'fb_type', 'gammatone', ...
+                'fb_lowFreqHz', 80, ...
+                'fb_highFreqHz', 8000, ...
+                'ihc_method', 'halfwave', ... % stream segr. uses 'dau'
+                'ild_wSizeSec', 20E-3, ...  % DnnLoc uses 20E-3, stream segr. uses 25E-3

```

Comment:
we can get rid of the comments here: the 25E-3 was due to Gabot filtering. Without it, identification happily uses 20E-3